examples/cartpole/cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py

```diff
@@ -128,7 +128,7 @@ def is_done(self):
         if abs(poleAngle) > 0.261799388:  # 15 degrees off vertical
             return True
 
-        cartPosition = round(self.robot.getPosition()[2], 2)  # Position on z axis
+        cartPosition = round(self.robot.getPosition()[0], 2)  # Position on x axis
```

Comment:
Change line 75~78 for `get_observations()`as well?
https://github.com/aidudezzz/deepworlds/blob/e9baef2bb658291020aed4ec28ad3a11f3ddfbd6/examples/cartpole/cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py#L75-L78

Reply:
Oops, nice catch! I will update it soon, thanks :)

Reply:
Update these comments as well. Thanks!
https://github.com/aidudezzz/deepworlds/blob/e9baef2bb658291020aed4ec28ad3a11f3ddfbd6/examples/cartpole/cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py#L22

https://github.com/aidudezzz/deepworlds/blob/e9baef2bb658291020aed4ec28ad3a11f3ddfbd6/examples/cartpole/cartpole_discrete/controllers/supervisor_manager/supervisor_controller.py#L42